src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.iOSmacOS.cs

```diff
@@ -13,10 +13,12 @@
 
 #if XAMARIN_IOS_UNIFIED
 using UIKit;
+using NativeView = UIKit.UIView;
 #elif XAMARIN_IOS
 using MonoTouch.UIKit;
 #elif __MACOS__
 using AppKit;
+using NativeView = AppKit.NSView;
```

Comment:
We now have a convention to prefix those aliases with an underscore: `_`

Reply:
Yes, I have switched to this in newer PRs, but forgot here, shall fix :-)